src/hkp.js

```diff
@@ -27,7 +27,7 @@ import config from './config';
  * Initialize the HKP client and configure it with the key server url and fetch function.
  * @constructor
  * @param {String}    keyServerBaseUrl  (optional) The HKP key server base url including
- *   the protocol to use e.g. https://pgp.mit.edu
+ *   the protocol to use e.g. https://keyserver.ubuntu.com
```

Comment:
```suggestion
 *   the protocol to use, e.g. 'https://pgp.mit.edu'; defaults to
 *   openpgp.config.keyserver (https://keyserver.ubuntu.com)
```

Reply:
I think its cool to leave the mit.edu one too...having options and choice is always good

Reply:
you don't think that it should be mentioned in the readme file also ??

Reply:
There are two examples in the readme, one mentions the Ubuntu server and one mentions the PGP server.